features/mbedtls/targets/TARGET_STM/TARGET_STM32F7/aes_alt.h

```diff
@@ -0,0 +1,503 @@
+/**
+ * \file aes_alt.h
```

Comment:
This file is very similar to `features/mbedtls/targets/TARGET_STM/TARGET_STM32L4/aes_alt.h`. It looks like L4 is just an older copy of aes.h from Mbed TLS (XTS is missing, for example, so the file is quite old). Can we move that L4 file so it is picked up and used for both targets instead of adding duplication?